go/private/actions/compilepkg.bzl

```
@@ -99,6 +99,7 @@ def emit_compilepkg(
         args.add("-cover_format", go.cover_format)
         args.add_all(cover, before_each = "-cover")
     args.add_all(archives, before_each = "-arc", map_each = _archive)
+    args.add("-label_name", go.label.name)
     if importpath:
         args.add("-importpath", importpath)
```

```suggestion
    if importpath:
        args.add("-importpath", importpath)
    else:
        args.add("-importpath", go.label.name)
```

How about this? So we don't need to add another option to compilepkg.go

---

@sluongng This PR looks almost ready to be merged except for this comment. I'm planning to prepare the 0.33.0 release soon and could include this fix.

---

Tbh, I am not quite sure on this suggestion.
It probably will work for the particular code block that I am interested in, but I am not sure if elsewhere in compilepkg.go do the code expect `importpath` variable to be empty in some rare scenarios.

As i did not have enough time to read through everything, I pushed up a fix according to the suggestion anyway.
I would trust CI result a lot more than myself. 😆

---

ok CI seems to be all green, I think this PR is good to merge @fmeum 🙏